ci/install.sh

```diff
@@ -19,6 +19,20 @@ install_standard_crates() {
     fi
 }
 
+install_system_dependencies() {
+    if [ $TARGET = x86_64-unknown-linux-musl ]; then
+        sudo apt-get install musl-tools
+        # download libunwind and build a static version w/ musl-gcc
+        wget https://github.com/libunwind/libunwind/releases/download/v1.3.1/libunwind-1.3.1.tar.gz
+        tar -zxvf libunwind-1.3.1.tar.gz
+        cd libunwind-1.3.1/
+        CC=musl-gcc ./configure --disable-minidebuginfo --enable-ptrace --disable-tests --disable-documentation
+        make
+        sudo make install
+        cd ..
+    fi
```

Comment:
With the latest remoteprocess, I don't think you will need to install libunwind - and since rbspy doesn't need this right now I think this block can be removed